site/tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "moduleResolution": "node",
     "sourceMap": true,
     "esModuleInterop": true,
+    // Need to disable this because generated typedefs omit generics which
+    // creates a problew for multiformats
+    // @see https://github.com/multiformats/js-multiformats#typescript-support
```

Comment:
This PR updates to v7

Reply:
> but we are on 4.2

Generated types don't have type parameters possibly because multiformats is either on older version of TS or maybe that bug has not been fixed yet.

Reply:
Oh 🤣 I just realised you meant typescript 4.2 - sorry ignore me

Reply:
I looked into this and seems that problem is not with multiformats, but rather with dag-cbor.